packages/evergreen-badges/src/styles/badge-appearances.js

```diff
@@ -0,0 +1,35 @@
+const baseStyle = {
+  boxSizing: 'border-box',
+  textTransform: 'uppercase',
+  WebkitFontSmoothing: 'antialiased',
+  WebkitAppearance: 'none',
+}
+
+const BadgeAppearances = {
+  neutral: {
+    ...baseStyle,
```

Comment:
I don't think we need a `BadgeAppearances` object.  `getBadgeStyle` is all we need in this case I think.